hawkbit-mgmt-resource/src/main/java/org/eclipse/hawkbit/mgmt/rest/resource/MgmtDistributionSetResource.java

```diff
@@ -137,10 +137,7 @@
 
     @Override
     public ResponseEntity<Void> deleteDistributionSet(@PathVariable("distributionSetId") final Long distributionSetId) {
-        final DistributionSet set = findDistributionSetWithExceptionIfNotFound(distributionSetId);
-
-        distributionSetManagement.deleteDistributionSet(set);
-
+        distributionSetManagement.deleteDistributionSet(distributionSetId);
```

Comment:
I like that you don't get an `EntityNotFoundException` anymore. Just du be shure is it intended?